crates/aranya-daemon/src/policy.md

```diff
@@ -128,6 +153,16 @@ fact DeviceSignKey[device_id id]=>{key_id id, key bytes}
 // A device's public EncryptionKey.
 fact DeviceEncKey[device_id id]=>{key_id id, key bytes}
 
+// A ID-based role on the team.
+fact Role[role_id id]=>{role struct RoleInfo}
```

Comment:
using `RoleInfo` here means we store the role ID twice. Please inline the fields here. Same with `Device` btw.